modules/update-preparing-conditional.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: PROCEDURE
+[id="update-preparing-conditional_{context}"]
+= Assessing the risk of conditional updates
+
+A conditional update is a supported update target that is available but not recommended due to a known risk that applies to your cluster.
```

Comment:
IMO we should remove the `supported` word from here. Just `A conditional update is an update path that is available but not recommended due to a known risk that applies to your cluster.`

Reply:
What RH supports is a hard question because we do not deny customers support in general irrespective of the situation.

Reply:
Sure thing, just wanted to take this opportunity to keep hammering the idea that "just because it doesn't have the word recommended next to it doesn't mean it's unsupported and has to be avoided at all costs", but I can remove the word here